tests/SpiffWorkflow/bpmn/ProcessDependencyTest.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+
+
+import sys
+import os
+import unittest
+
+from SpiffWorkflow.camunda.parser.CamundaParser import CamundaParser
+from SpiffWorkflow.dmn.parser.BpmnDmnParser import BpmnDmnParser
+from SpiffWorkflow.spiff.parser import SpiffBpmnParser
+from tests.SpiffWorkflow.bpmn.BpmnLoaderForTests import TestBpmnParser
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', '..', '..'))
+from SpiffWorkflow.bpmn.workflow import BpmnWorkflow
+from tests.SpiffWorkflow.bpmn.BpmnWorkflowTestCase import BpmnWorkflowTestCase
+
+__author__ = 'danfunk'
+
+
+class ProcessDependencyTest(BpmnWorkflowTestCase):
+    """
+    Assure we can determine all of the call activities and DMN references that
+    will be required by a parser, prior to calling its parse method.
+
+    Because DMN references vary between Camunda and Spiff, need to test that
+    both methods will work.
+    """
+
```

Comment:
For the record, I don't like camunda and spiff appearing here, though given the way our test suite is set up, there probably isn't a better way.